frontend/src/libs/components/order-list-card/libs/helpers/status-converter/status-converter.helper.ts

```diff
@@ -0,0 +1,31 @@
+import { OrderStatus } from 'shared/build';
+import { type OrderStatusValues } from 'shared/build/packages/orders/libs/types/order-status-values.type';
+
+import { type Color } from '~/libs/types/color.type';
+
+const statusConverter = (
+  status: OrderStatusValues,
+): { name: string; color: Color } => {
+  switch (status) {
+    case OrderStatus.PENDING: {
+      return { name: 'Pending', color: 'grey-light' };
+    }
+    case OrderStatus.CONFIRMED: {
+      return { name: 'On the way', color: 'blue-light' };
+    }
+    case OrderStatus.CANCELED: {
+      return { name: 'Canceled', color: 'red' };
+    }
+    case OrderStatus.DONE: {
+      return { name: 'Done', color: 'grey-dark' };
+    }
+    case OrderStatus.PICKING_UP: {
+      return { name: 'Arrived', color: 'green' };
+    }
+    default: {
+      return { name: 'Pending', color: 'grey-light' };
```

Comment:
I think we should add additional var to the palette. For example for canceled badges you used the color "#ee2a64" but in our layout  we have "#ee2a6426" the same for green

Reply:
We can use $red as a base color, and derive a required transparent variation of it by using [sass color functions](https://sass-lang.com/documentation/modules/color/). There is a working example in frontend\src\assets\css\plugins\balloon-css.plugin.scss

Reply:
I suggest use the existed palette.